### DISCUSSION TITLE
FOXL2 pathogenic range check

### DISCUSSION THREAD

**Discussion by hdashnow** [dashnowlab/STRchive#37] [Locus updates and new loci]
This locus differs between gnomAD, STRipy and STRchive. 
https://link.springer.com/article/10.1007/s00439-006-0276-0

See also https://omim.org/entry/110100#molecularGenetics

**Comment by laurelhiatt** [comment]
I think this is a phenotypic spectrum thing. Our normal ranges are the same, so it's about where you're considering the beginning of pathogenicity. I think this will be resolved by our fully penetrant, intermediate range changes
